dht/token_range_endpoints.hh

```diff
@@ -24,10 +24,11 @@
 #include <seastar/core/sstring.hh>
 
 #include "seastarx.hh"
+#include "gms/inet_address.hh"
 
 namespace dht {
 struct endpoint_details {
-    sstring _host;
+    gms::inet_address _host;
```

Comment:
This change should not have been squashed with this commit. It affects existing subsystems, and is a preparatory transformation which can stand on its own.

Reply:
Done.